### ISSUE TITLE
Excessive whitespace on tablets obscures bookmarks

### ISSUE THREAD

**Issue by o8k57** [maxpower47/PinDroid#51]
On the Nexus 9 tablet, bookmarks are squeezed into a tiny portion of the screen, obscuring text and wasting most of the available space.

Portrait and landscape screenshots attached.

![screenshot_20160428-220612](https://cloud.githubusercontent.com/assets/19149667/14954357/a9c05ac0-1069-11e6-9055-fbf8b0c382e3.png)
![screenshot_20160502-125602](https://cloud.githubusercontent.com/assets/19149667/14954358/a9c0d626-1069-11e6-8187-39883b03a88f.png)

**Comment by karlicoss** [comment]
@o8k57 agree, I am annoyed by it as well. I could take a look and fix that, but I'm not a very good designer, so what would you suggest instead? (since stretching the list to fit the screen width would not look good either).
Maybe we could at least just make it centered and wider in landscape mode.

**Comment by PatrickHonorez** [comment]
Same issue on a 10" Lenovo Yoga Pro where the list uses only 1/4 of the screen width. 
I suggest stretching the list to screen width. At least it will provide full functionality.

**Comment by vitriolix** [comment]
![screenshot_20171202-202517](https://user-images.githubusercontent.com/7844/33522333-088919fe-d79f-11e7-9c3c-e6e2dababdf4.png)

The standard is to show the navigation drawer items on the left panel to prevent wasted white space, maybe I'll take a crack at it
